deepspeed/module_inject/tp_shard.py

```diff
@@ -20,16 +20,13 @@ def get_num_kv_heads():
 def get_shard_size(total_size, mp_size, rank=None):
     global num_kv_heads
     # When we have num_kv_heads defined, uneven division is possible, otherwise enforce even division
```

Comment:
change 'otherwise enforce even division' to 'otherwise enforce near even division'